domestic/cms_panels.py

```diff
@@ -32,6 +32,11 @@ class GreatDomesticHomePagePanels:
                 FieldPanel('hero_cta_url'),
             ],
         ),
+        MultiFieldPanel(
+            heading='Magna CTAs',
```

Comment:
Minor: naming fields is hard, so magna_ctas is probably fine for these new CTAs, but it might be good to tweak the heading - eg
```suggestion
            heading='Top-of-page CTAs',
```